specification/components/schemas/responses/retrieveOboPractitioners/400Response.yaml

```diff
@@ -0,0 +1,21 @@
+description: |
+  Where status code 400 (bad request error) is returned then an eRS-OperationOutcome-1 will be included in the body, as detailed below. 
+  Check diagnostics property for specific information regarding the error.
+  
+  | Error code                         | Description                                                                          |
+  | ---------------------------------- | ------------------------------------------------------------------------------------ |
+  | BAD_REQUEST                        | The mandatory _query parameter of onBehalfOf is not present.                         |
```

Comment:
I think we need to confirm what the error code should be in this scenario. Following the BaRS specification I think this would be `REC_BAD_REQUEST` but that needs to be confirmed.